kong/init.lua

```diff
@@ -504,6 +500,12 @@ function Kong.ssl_certificate()
 
   runloop.certificate.before(ctx)
 
+  local ok, err = plugins_map_wrapper()
+  if not ok then
+    ngx_log(ngx_CRIT, "could not ensure plugins map is up to date: ", err)
+    return responses.send_HTTP_INTERNAL_SERVER_ERROR()
```

Comment:
This will be trouble in this context (`certificate`). Mind following the same pattern as we do for this phase? See: https://github.com/Kong/kong/blob/0.14.1/kong/runloop/certificate.lua#L106-L107

Reply:
whups! thanks!